Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/View.purs

```diff
@@ -279,10 +279,30 @@ view push state =
             , if state.props.emergencyHelpModal then (emergencyHelpModal push state) else emptyTextView state
             , if state.props.showShareAppPopUp then (shareAppPopUp push state) else emptyTextView state
             , if state.props.showMultipleRideInfo then (requestInfoCardView push state) else emptyTextView state
+            , if state.props.showLiveDashboard then showLiveStatsDashboard push state else emptyTextView state
```

Comment:
add the version check here as well

Reply:
We check the version and hide the live_stats view so no need to add version check here as well